src/js/routes/services.js

```diff
@@ -1,37 +1,69 @@
 import Router from 'react-router';
-let Route = Router.Route;
+let {Route, Redirect} = Router;
 
+import DeploymentsTab from '../pages/services/DeploymentsTab';
 import ServiceOverlay from '../components/ServiceOverlay';
 import ServicesPage from '../pages/ServicesPage';
+import ServicesTab from '../pages/services/ServicesTab';
 
-let servicesRoutes = {
+let services = {
   type: Route,
   name: 'services',
-  path: 'services/?',
   handler: ServicesPage,
   children: [
     {
       type: Route,
-      name: 'services-detail',
-      path: ':id/?'
+      name: 'services-page',
+      path: '/services/?',
+      children: [
+        {
+          type: Route,
+          name: 'services-all',
+          path: 'all/',
```

Comment:
@philipnrmn can we make the routes work like this:
`/services` <-- All services list
`/services/:id` <-- Detail page of a service
`/services/:id/ui` <-- Opens the service overlay and the UI for the service

I have some comments about the rest of the URLs, but non blockers for this PR.